chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SchedulerManagerImpl.java

```diff
@@ -166,20 +197,34 @@ public ChplTrigger updateTrigger(final ChplTrigger trigger) throws SchedulerExce
      * will need to be added to the list.
      */
     @Override
-    @PreAuthorize("hasAnyRole('ROLE_ADMIN','ROLE_ONC', 'ROLE_ACB')")
+    @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).SCHEDULER, "
+            + "T(gov.healthit.chpl.permissions.domains.SchedulerDomainPermissions).GET_ALL)")
+    //@PostFilter("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).SCHEDULER, "
+    //        + "T(gov.healthit.chpl.permissions.domains.PendingSurveillanceDomainPermissions).GET_ALL, filterObject)")
     public List<ChplJob> getAllJobs() throws SchedulerException {
         List<ChplJob> jobs = new ArrayList<ChplJob>();
         Scheduler scheduler = getScheduler();
+
+        //Get all the jobs (no security)
         for (String group : scheduler.getJobGroupNames()) {
-            if (group.equals("chplJobs")) {
-                for (JobKey jobKey : scheduler.getJobKeys(groupEquals(group))) {
-                    JobDetail jobDetail = scheduler.getJobDetail(jobKey);
-                    if (doesUserHavePermissionToJob(jobDetail)) {
-                        jobs.add(new ChplJob(jobDetail));
-                    }
-                }
+            for (JobKey jobKey : scheduler.getJobKeys(groupEquals(group))) {
+                JobDetail jobDetail = scheduler.getJobDetail(jobKey);
+                jobs.add(new ChplJob(jobDetail));
+            }
+        }
+
+        //TODO: There is a problem using @PostFilter - some sort of ACL issue
+        //Until that is figured out...
+        Iterator<ChplJob> iterator = jobs.iterator();
```

Comment:
I thought you had a problem with PostFilter for the surveillance stuff but figured it out. Am I remembering that wrong or is this a different problem?

Reply:
I completely forgot!  I'll update this to work with @PostFilter.